continuationsPlugin/src/test/scala/continuations/CallsSuspendParameterSuite.scala

```diff
@@ -12,7 +12,9 @@ class CallsSuspendParameterSuite extends FunSuite, CompilerFixtures {
       val call = tree match
         case Block(_, call) => call
 
-      assertNoDiff(CallsSuspendParameter.unapply(call).get.show, call.show)
+      val prev = call.show
+      val _ = CallsSuspendParameter(call)
+      assertNoDiff(call.show, prev)
```

Comment:
probably should change this one to check it is `true`

Reply:
after discussing with @diesalbla it seems that some of these predicates change the tree in a way that affects `.show`.
for example [here](https://github.com/47deg/TBD/blob/main/continuationsPlugin/src/test/scala/continuations/HasSuspensionNotInReturnedValueSuite.scala#L33) if we do the same comparison as above we see this difference
```
=> Diff (- obtained, + expected)
-val mySuspend: ((continuations.Suspend : Object {...})) ?=> Int = ∙
+val mySuspend: (continuations.Suspend.type) ?=> Int = ∙
```

the trees though before and after are the same, if we don't use `.show`. 

since this seems to not affect our transformation and we use these predicates only to see if they return true/false I think this is irrelevant for now and if we want to investigate it more then should do separately